__tests__/intergration.test.js

```diff
@@ -33,3 +33,16 @@ describe('GET api', () => {
       });
   });
 });
+
+describe('GET /api/articles/:article_id', () => {
+    test('get 200: should return an article as specified by its ID', () => {
+        const output = [{"article_id": 2, "article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700", "author": "icellusedkars", "body": "Call me Mitchell. Some years ago—never mind how long precisely—having little or no money in my purse, and nothing particular to interest me on shore, I thought I would buy a laptop about a little and see the codey part of the world. It is a way I have of driving off the spleen and regulating the circulation. Whenever I find myself growing grim about the mouth; whenever it is a damp, drizzly November in my soul; whenever I find myself involuntarily pausing before coffin warehouses, and bringing up the rear of every funeral I meet; and especially whenever my hypos get such an upper hand of me, that it requires a strong moral principle to prevent me from deliberately stepping into the street, and methodically knocking people’s hats off—then, I account it high time to get to coding as soon as I can. This is my substitute for pistol and ball. With a philosophical flourish Cato throws himself upon his sword; I quietly take to the laptop. There is nothing surprising in this. If they but knew it, almost all men in their degree, some time or other, cherish very nearly the same feelings towards the the Vaio with me.", "created_at": "2020-10-16T05:03:00.000Z", "title": "Sony Vaio; or, The Laptop", "topic": "mitch", "votes": 0}]
+        return request(app)
+        .get('/api/articles/2')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.article.length).toBe(1)
```

Comment:
There should be no need for a length check as the article should npt be returned in an array. Instead the article object should directly be returned on a key of something like `article`. You should take care of this "shaping of the data to be returned" in the model so the controller can simply return it as above.